packages/fxa-react/styles/ctas.css

```diff
@@ -20,8 +20,9 @@
       @apply border-grey-400 bg-grey-100 text-grey-400;
     }
 
-    &:focus {
-      @apply bg-white;
+    &:focus,
+    &:focus-visible {
+      @apply bg-white outline-2 outline-offset-2 outline-blue-500;
```

Comment:
Unsure if this should be a blue-500 outline or a grey outline of some sort.

Reply:
@dschom I'm still seeing a system color outline on the focus states here:

<img width="472" alt="image" src="https://user-images.githubusercontent.com/13018240/184446612-b597eddb-caa9-47a8-b702-6bcd3e43efc0.png">

<img width="158" alt="image" src="https://user-images.githubusercontent.com/13018240/184446677-833eca7e-35bd-43d4-866a-5f1063bc8b3b.png">

I ran into this when working on my currently open PR with checkboxes and was headdesking for a bit until I figured it out - I don't know why, but we need to set `outline-style: solid` on them which can be done by adding the `outline` class. I wonder if you haven't changed your system default color and that somehow makes a difference or something? Not sure.

Reply:
@LZoog I definitely don't experience this, and can't seem to reproduce. If I inspect the element, I see the correct focus ring coming through.  

Just double checking but did you rebuild? You can see the blue color is 2px and the same color as specified by tail wind. I can add the style override too, but would be curious to know why I don't see this and you do. Here's a screen shot of my dev tools and an neutral button:

![image](https://user-images.githubusercontent.com/94418270/184453896-bee3134e-e192-45a5-b933-b2a639e5aeb2.png)

I also checked my config settings... None of my defaults are altered. I wonder if yours are, and mabye this is why we see different things?
![image](https://user-images.githubusercontent.com/94418270/184453652-0c56ac73-8d7a-43b4-a8b4-b7388620f06b.png)

And as you can see if I select 'show only modified preferences' none of mine are modified.
![image](https://user-images.githubusercontent.com/94418270/184453670-cfb28671-83b7-4e2e-b521-5fff1ef810b0.png)

Reply:
@LZoog btw, I just added the 'outline' class and now see the outline-style:solid is defined. Let me know if this fixes it for you.

Reply:
For future readers since I'm r+'ing, Dan and I DM'd a bit about this and he was able to see what I pointed out here after changing system pref highlight color to something other than blue. Not sure if choosing a different color other than the default takes precedence for some reason and needs an override with `outline` or if it was just more obvious picking a default color that wasn't a similar shade of blue but either way, 👍